### PR TITLE
Add CCE addon support

### PR DESCRIPTION
**Pull request by outcatcher**
## Summary of the Pull Request
Implement CCE addon installation using new resource `resource_opentelekomcloud_cce_addon_v3`
Implement #536

## PR Checklist

* [x] Refers to: #536
* [x] Tests added/passed.
* [x] Documentation updated.

## Acceptance Steps Performed
```
=== RUN   TestAccCCEAddonV3_basic
--- PASS: TestAccCCEAddonV3_basic (407.09s)
PASS

Process finished with exit code 0
```

### CODE REVIEW

**Comment by outcatcher** [comment]
recheck

**Comment by outcatcher** [comment]
recheck

**Comment by outcatcher** [comment]
OTC issue fixed: 
https://jira.tsi-dev.otc-service.com/browse/OCS-2690

**Comment by outcatcher** [comment]
recheck

File: docs/resources/cce_addon_v3.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Example Usage
+
+```hcl
+variable "flavor_id" { }
+variable "vpc_id" { }
+variable "subnet_id" { }
+
+resource opentelekomcloud_cce_cluster_v3 cluster_1 {
+  name                    = "%s"
```
Comment (lego963) [inline docs/resources/cce_addon_v3.md]: strange name, change pls

File: docs/resources/cce_addon_v3.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  template_name    = "metrics-server"
+  template_version = "1.0.3"
+  cluster_id       = opentelekomcloud_cce_cluster_v3.cluster_1.id
+
+  values {
+    basic = {
+      euleros_version = "2.5"
+      rbac_enabled    = true
+      swr_addr        = "100.125.7.25:20202"
+      swr_user        = "hwofficial"
```
Comment (lego963) [inline docs/resources/cce_addon_v3.md:36]: can we change user name to something else?

File: opentelekomcloud/resource_opentelekomcloud_cce_addon_v3.go [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"template_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Computed: true,
```
Comment (lego963) [inline opentelekomcloud/resource_opentelekomcloud_cce_addon_v3.go:35]: I suppose that we need to validate this argument both with `template_name`
If it's possible

File: opentelekomcloud/resource_opentelekomcloud_cce_node_v3.go [hunk 4]
```diff
@@ -366,6 +366,7 @@ func resourceCCENodeV3Create(d *schema.ResourceData, meta interface{}) error {
 		Spec: nodes.Spec{
 			Flavor:      d.Get("flavor_id").(string),
 			Az:          d.Get("availability_zone").(string),
+			Os:          "EulerOS 2.5",
```
Comment (lego963) [inline opentelekomcloud/resource_opentelekomcloud_cce_node_v3.go]: why did you change this file?

**Review by lego963 — CHANGES_REQUESTED** [review]

**Review by lego963 — APPROVED** [review]

**Pull request merged**